src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/CSharpLSPSnippetTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Completion.CompletionProviders.Snippets;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities.Snippets;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionProviders.Snippets
+{
+    public class CSharpLSPSnippetTests : AbstractCSharpLSPSnippetTests
```

Comment:
same with this one

Reply:
what's the difference between CSharpLSPSnippetTests  and AbstractCSharpLSPSnippetTests?  why shouldn't they be merged?  What other subclasses of AbstractCSharpLSPSnippetTests do you have/expect?

Reply:
this comment still applies.